spec/support/vcr_cassettes/bgs/power_of_attorney_request/proc_create.yml

```diff
@@ -0,0 +1,243 @@
+---
```

Comment:
we could point to the [existing cassette](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr_cassettes/bgs/form674/submit.yml) for the specs, if you'd like to reuse that

Reply:
I stand correct, that's a formCreate.. but [here is a procCreate](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr_cassettes/bgs/service/create_proc.yml).. I believe it only matches method and uri, so we should be able to utilize existing cassettes if we'd want to.

Reply:
TIL VCR doesn't match on body by default. I like making our own cassettes here just in case bgs-ext changed their code and having cassettes that match our configuration will provide a better story/history for future devs as bgs local evolves.